src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/ReleaseController.cs

```diff
@@ -30,7 +30,7 @@ public async Task<ActionResult<List<ReleaseSummaryViewModel>>> ListReleases(stri
         [HttpGet("publications/{publicationSlug}/releases/latest")]
         public async Task<ActionResult<ReleaseViewModel>> GetLatestRelease(string publicationSlug)
         {
-            return await _releaseService.Get(publicationSlug)
+            return await _releaseService.GetCachedViewModel(publicationSlug)
```

Comment:
Yeh I'd personally say we shouldn't be prefixing stuff with 'cached'. Caching is an implementation detail and naming methods like this would be quite limiting if we had different implementations of the underlying interfaces.

That said, we're not exactly running an OOP purity class here, so if people are finding it problematic to know whether a method is cached or not, we can go with it.